### ISSUE TITLE
hostname instead of ip by logging

### ISSUE THREAD

**Issue by 7c** [novnc/websockify#392]
Hi,

thanks for this wonderful product. HAve following suggestion:

At given scenario:
```
WebSocket server settings:
  - Listen on :6080
  - Flash security policy server
  - No SSL/TLS support (no cert file)
  - Backgrounding (daemon)
  - proxying from :6080 to 127.0.0.1:5901
ec2-18-207-218-7.compute-1.amazonaws.com - - [18/Apr/2019 20:01:09] 18.207.218.7: Plain non-SSL (ws://) WebSocket connection
ec2-18-207-218-7.compute-1.amazonaws.com - - [18/Apr/2019 20:01:09] 18.207.218.7: Version hybi-13, base64: 'False'
ec2-18-207-218-7.compute-1.amazonaws.com - - [18/Apr/2019 20:01:09] 18.207.218.7: Path: '/websockify'
ec2-18-207-218-7.compute-1.amazonaws.com - - [18/Apr/2019 20:01:09] connecting to: 127.0.0.1:5901
```

i have started it with -v and --log-file. As you see clients host is shown as hostname instead of ip. 

Imagine this: after weeks this host might resolve to another ip address behind the scenes. Especially if someone is logging for abuse purposes the hostname is not really useable because it is unreliable. Having timestamp and ip address would be more appropriate for many use cases.

May i suggest to switch to ip from host to log ?

thanks for listening

**Comment by samhed** [comment]
Sounds good to me.

**Comment by CendioOssman** [comment]
An ip address is also the default behaviour from the Python library. Not sure why we decided to do things differently...

**Comment by 7c** [comment]
Another issue with hostnames might be the resolve time. We are dealing with hostnames and dns a lot. We know that it might take up to 5 seconds to resolve an ip (which is probably your low level identifier) to a hostname. Many ip addresses do not have a reverse resolve capability. Thanks for considering

**Comment by CendioOssman** [comment]
Fixed in be9823bf938a9ebce65572941651973dbaa366f8. Thanks for reporting.
